src/clj/rems/db/blacklist.clj

```diff
@@ -39,6 +36,10 @@
   (s/validate BlacklistEvent event)
   (json/generate-string event))
 
+(defn- event-from-db [event]
+  (assoc (json->event (:eventdata event))
+         :event/id (:event/id event)))
+
 (defn- check-foreign-keys [event]
```

Comment:
as I mentioned in my previous review, this function could be on the service level to reduce the dependencies of the db level. not insisting you move it though.

Reply:
Overlooked that, sorry. Yeah, that's a good idea. Done.

Reply:
Okay hm, it seems that part of the tests (namely, test-parameter-validation), currently located in rems.db.test-blacklist, would then have to be moved to rems.api.services.test-blacklist, which doesn't exist currently. I think it's cleaner to have all the tests in one place, and it's also good to have the check close to the database level, especially since we might want to eventually do that in the database (see the comment in check-foreign-keys function).

For these reasons, I ended up keeping the function where it is in this patch. Thanks for the suggestion, though.